appoptics.go

```diff
@@ -31,10 +31,11 @@ type Reporter struct {
 	WhitelistedRuntimeMetrics map[string]bool        // runtime.* metrics to upload (nil = allow all)
 	TimerAttributes           map[string]interface{} // units in which timers will be displayed
 	intervalSec               int64
+	measurementsURI           string
 }
 
 func NewReporter(registry metrics.Registry, interval time.Duration, token string, tags map[string]string,
-	percentiles []float64, timeUnits time.Duration, prefix string, whitelistedRuntimeMetrics []string) *Reporter {
+	percentiles []float64, timeUnits time.Duration, prefix string, whitelistedRuntimeMetrics []string, measurementsURI string) *Reporter {
```

Comment:
see below on defaulting for the measurementsURI, also in this case this is a method signature change too. The only alternative in this case would be to have a net-new NewReporterWithMeasurementsURI method though, so I guess just rolling with a breaking change (since this is still a pre-1.0 release) is an OK way to go.